### ISSUE TITLE
PoolNBEATS Loss error + quantile error

### ISSUE THREAD

**Issue by GaetanDu** [AutocoinLab/autopycoin#19]
```python
from autopycoin.models import PoolNBEATS, create_interpretable_nbeats, NBEATS, Stack
from autopycoin.layers import TrendBlock, SeasonalityBlock
from tensorflow.keras import Model
from autopycoin.losses import QuantileLossError , quantile_loss
from autopycoin.utils import quantiles_handler

best_hps = tuner.get_best_hyperparameters(num_trials = 1)[0]

w = WindowGenerator(
        input_width=91,
        label_width=91,
        shift=91,
        test_size=20,
        valid_size=40,
        flat=False,
        batch_size=64
    )

w = w.from_array(data=data.interpolate(method='time')[[0, 1, 2]].values)

class CPoolNBEATS(PoolNBEATS):
    def __init__(self, **kwargs:dict):
        super().__init__(**kwargs)
    def predict(self, *args:list, **kwargs:dict):
        outputs = super().predict(*args, **kwargs)
        return tf.where(outputs > 0, outputs, 0)
    
class Hierarchical(tf.keras.losses.Loss):
    def call(self, y_true, y_pred):
        loss = self.get_loss_hierarchical(y_true, y_pred)
        return loss
    
    def get_loss_hierarchical(self, y_true, y_pred):
        pred = tf.reduce_sum(y_pred, axis=-1)
        true = tf.reduce_sum(y_true, axis=-1)
        return tf.reduce_mean(tf.math.square(pred - true), axis=-1)
    
class QuantileLossError(tf.keras.losses.Loss):
    def __init__(self, quantiles, **kwargs):
        super().__init__(**kwargs)
        self.quantiles = quantiles_handler(quantiles)
    def call(self, y_true, y_pred):
        loss = quantile_loss(self.quantiles, y_true, y_pred)
        return loss
                 
nbeats = lambda: create_interpretable_nbeats(
    label_width=91,
    p_degree=best_hps.get('p_degree'),
    forecast_periods=91,
    trend_n_neurons=best_hps.get('neurons_trend'),
    seasonality_n_neurons=best_hps.get('neurons_seas'),
    drop_rate=0,
    share=True)
    
model = CPoolNBEATS(
                 n_models=10,
                 nbeats_models=nbeats,
                 losses=[QuantileLossError([0.2, 0.5, 0.8])])
model.compile(tf.keras.optimizers.Adam(
    learning_rate=0.0001, beta_1=0.9, beta_2=0.999, epsilon=1e-07, amsgrad=True,
    name='Adam'), loss=model.get_pool_losses(), metrics=['mae'])

history = model.fit(w.train, validation_data=w.valid, epochs=25)
```

**Comment by GaetanDu** [comment]
branch 19: Done
